rpi/src/platform_rpi.h

```diff
@@ -3,3 +3,24 @@
 #include "platform.h"
 
 void processNetworkQueue();
+
+#define NUM_WORKERS 3
+
+class RPiPlatform : public Platform {
```

Comment:
I only made Raspberry Pi instantiate a `LinuxPlatform` directly.

Updated in: https://github.com/tangrams/tangram-es/pull/1263/commits/63e5a18585b28b8cce9debfa56880cb1713cadc3